### PR TITLE
fixed the warning to do with the use of <h2> with material ui, also the…

### PR DESCRIPTION
**Pull request by valzugg**
fixed the warning to do with the use of <h2> with material ui, also the footer now stays centered. Could not resolve the other warning

### CODE REVIEW

File: src/pages/ThankYouPage.jsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
 
-const ThankYouPage = () => (
+const ThankYouPage = () => ( //TODO: attributes of class txt should be implemented with material UI
     <div>
         <div className="center-align"><img src="/planblogo_color.jpg" className="logo"></img> </div>
         <div className="answer-div">
-            <Typography class="txt"><h2>Kiitos palautteesta!</h2></Typography>
+            <Typography class="txt" variant="h2">Kiitos palautteesta!</Typography>
```
Comment (VonLatvala) [inline src/pages/ThankYouPage.jsx]: saako tuota "class" käyttää noin? Eikö pitäisi olla className ?

Comment (valzugg) [reply src/pages/ThankYouPage.jsx]: that is the other problem that i could not fix, its coming from a css file, changing it to className removes all the styling

Comment (VonLatvala) [reply src/pages/ThankYouPage.jsx]: then it's probably supposed to be like that

**Comment by VonLatvala** [comment]
image
Does not work on iPad Pro (styling, that is), but maybe it isn't our target platform?

**Comment by khenriksson** [comment]
Style is off because of material.io, and if I remember correctly the variant="h2" fucks up the styling which is why I didn't apply it in the first place

**Pull request merged**